about.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <link href="style/style_about.css" rel="stylesheet"/>
+  <link href="normalize.css" rel="stylesheet"/>
+  <meta charset="utf-8">
+  <title>About</title>
+</head>
+<body>
+  <header><h2>About</h2></header>
+
+  <section class="sect1">
+    <img src="images/Semret for Ada.jpg" alt="Photo of Semret" />
+    <div class="parg1"> Self-driven, quick learner and innovative professional with over 5 years comprehensive experience in global and domestic (US) healthcare systems, project management, program development, implementation and management, research methodologies, data analyses and manuscript publication.
+    </div>
+  </section>
+
+  <section class="sect2">
+    <div class="parg2">Currently, I am an Ada Developers Academy Student for C10 cohort and loving it. In the near future, I see myself working for a large tech company as a full stack developer. I want a career that is as challenging as it is exciting, but one that also makes life easier for people. Adding to the solid programming skills I will be gaining from Ada, I plan to keep the learning spirit and expand my knowledge with practical experience to develop a better understanding of all the layers of software development.
+    </div>
+  </section>
```

Comment:
You should pick more meaningful class names than `sect1` or `sect2`. Think of them the same way you would variable names.